pkg/models/system_intake_grb_reviewers.go

```diff
@@ -33,6 +34,41 @@ const (
 	SIGRBRVRAlternate SIGRBReviewerVotingRole = "ALTERNATE"
 )
 
+func (r SIGRBReviewerVotingRole) Humanize() string {
+	var grbVotingRoleTranslationsMap = map[SIGRBReviewerVotingRole]string{
+		SIGRBRVRVoting:    "Voting",
+		SIGRBRVRNonVoting: "Non-voting",
+		SIGRBRVRAlternate: "Alternate",
+	}
+	translation, ok := grbVotingRoleTranslationsMap[r]
+	if !ok {
+		panic(fmt.Errorf("%s is not a valid SIGRBReviewerVotingRole", r))
+	}
+	return translation
+}
+
+func (r SIGRBReviewerRole) Humanize() string {
+	var grbRoleTranslationsMap = map[SIGRBReviewerRole]string{
+		SIGRBRRCoChairCIO:          "Co-Chair - CIO",
+		SIGRBRRCoChairCFO:          "CO-Chair - CFO",
+		SIGRBRRCoChairHCA:          "CO-Chair - HCA",
+		SIGRBRRACA3021Rep:          "ACA 3021 Rep",
+		SIGRBRRCCIIORep:            "CCIIO Rep",
+		SIGRBRRProgOpBDGChair:      "Program Operations BDG Chair",
+		SIGRBRRCMCSRep:             "CMCS Rep",
+		SIGRBRRFedAdminBDGChair:    "Fed Admin BDG Chair",
+		SIGRBRRProgIntBDGChair:     "Program Integrity BDG Chair",
+		SIGRBRRQIORep:              "QIO Rep",
+		SIGRBRRSubjectMatterExpert: "Subject Matter Expert (SME)",
+		SIGRBRROther:               "Other",
+	}
+	translation, ok := grbRoleTranslationsMap[r]
+	if !ok {
+		panic(fmt.Errorf("%s is not a valid SIGRBReviewerRole", r))
```

Comment:
Good catch, I left the panic in because I wanted to discuss the best way to handle this with you and @ClayBenson94!

Reply:
we can declare that map outside of the function btw

Reply:
maybe just default to "Other"?

Reply:
The function is the only thing using the map is the only reason I placed it inside.

As far as "Other" or "". I was torn between avoiding returning an error and returning a non-value since the only time that would happen is if someone purposefully created a value that wasn't from the enum which shouldn't happen, but technically could. 

I guess the Go way would be to just suck it up and do the error checks?